functions-methods.js

```diff
@@ -33,4 +55,18 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email){
+    let emailIncludes = "";
+
+    if(email.includes("@"&&".nl") || email.includes("@"&&".com")){
+        emailIncludes = "true";
+    }else {
+        return emailIncludes = "false";
+    }
+
+    return emailIncludes;
```

Comment:
En uiteindelijk kun je het dan korter schrijven
```suggestion
    return email.includes("@") && (email.includes(".nl") || email.includes(".com"));
```